DXRFixes/DeusEx/Classes/Lamp1.uc

```diff
@@ -0,0 +1,6 @@
+class DXRLamp1 injects Lamp1;
+
+defaultproperties
+{
+     LightRadius=16
```

Comment:
Why are we doubling the light radius from a table lamp?  From the image you had posted, the light radius on the table lamp looks about right for a typical lamp - illuminating the area, not the whole room

![image](https://github.com/Die4Ever/deus-ex-randomizer/assets/13684088/e25763b4-b935-44c1-86a5-9418488b0e5a)

Reply:
This is what it looks like with Lamp1 and Lamp2 doubled.
![image](https://github.com/Die4Ever/deus-ex-randomizer/assets/17994307/b726c407-19c7-46e9-b92a-d5ea1d2d69bd)
The table lamp is still just illuminating a relatively small area around it. A real table lamp definitely illuminates more than they do in vanilla. (That's true for all three lamps really.) I wouldn't be opposed to turn Lamp1 down to maybe 1.5X vanilla.

Reply:
This is the (probably) final look.
![image](https://github.com/Die4Ever/deus-ex-randomizer/assets/17994307/e589cfeb-003e-47a8-9b30-207717a6336f)